tools/External Tools.xml

```diff
@@ -126,18 +126,25 @@
       <option name="WORKING_DIRECTORY" value="$ProjectFileDir$/.." />
     </exec>
   </tool>
+  <tool name="Insert and verify test id(s) Current" description="Insert and verify the unique test id" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="false" showConsoleOnStdErr="false" synchronizeAfterRun="true">
```

Comment:
@MichaelOrlov Can you remove this line? Because we will not use the test id tool anymore after the sphinx-needs are working.

Reply:
@dongilchang But how is our CI currently running? Does it still check for the test UIDs?

Reply:
@dongilchang Ok. I removed the button for TestIDs from the panel but still will keep the action item for it in the External tools toolbox.
The toolbox panel is going to look like 
![image](https://github.com/ApexAI/ApexAI_clion_settings/assets/7294722/88c1ab5a-d5c8-41b5-ab0f-16415ffc876f)